src/BoilerPlate/Game.cpp

```diff
@@ -55,25 +56,38 @@ void Game::Update(float currentHeight, float currentWidth, float deltaTime)
 
 void Game::Render()
 {
+	//===========================================================================
 	glClearColor(colors.slateBlue.redValue, colors.slateBlue.greenValue,
 		colors.slateBlue.blueValue, colors.slateBlue.alphaValue);
 	glClear(GL_COLOR_BUFFER_BIT);
+	//===========================================================================
 
 
+	//===========================================================================
```

Comment:
Honestly, don't like this "separators" but if you think they help the readability, go for it.